### ISSUE TITLE
MPD- body composition terms

### ISSUE THREAD

**Issue by obophenotype-user** [obophenotype/mammalian-phenotype-ontology#1569]
Molly,

New terms/ids for this batch:

abnormal body fat mass MP:0012320
abnormal total tissue mass MP:0012321
decreased total tissue mass MP:0012322
increased total tissue mass MP:0012323

New hierarchy:

Cindy

From: Molly Bogue molly.bogue@jax.org
Date: Monday, November 18, 2013 12:28 PM
To: Cynthia Smith cynthia.smith@jax.org
Subject: MPD requesting MP terms

Hi Cindy,

We need new terms to cover body composition.  

Currently MP has 'abnormal lean body mass' and 'abnormal percent body fat', but they are not even in the same MP branch.  

I am proposing we bring these together under a new term called 'abnormal body composition' and keep the other relationships in place, taking advantage of the DAG.

To best understand what I'm trying to convey, take a look at the attached file (worksheet 1).  You can see the terms, hierarchy, and places I propose we need new or multiple parents.

The second worksheet sums up what we need to add to accomplish this (new terms and new parent relationships).

This is kind of convoluted.  I am certainly open to suggestions.  The bottom line, though, is that we don't have terms that cover fat mass (as opposed to percent body fat) and total tissue mass.  These are very common measurements for body composition.

Let me know if you have any questions.

Thanks
Molly

Reported by: cindyJax

Original Ticket: [obo/mammalian-phenotype-requests/1569](https://sourceforge.net/p/obo/mammalian-phenotype-requests/1569)
